src/Compiler/Checking/CheckPatterns.fs

```diff
@@ -441,7 +441,7 @@ and TcPatArrayOrList warnOnUpper cenv env vFlags patEnv ty isArray args m =
 
 and TcRecordPat warnOnUpper cenv env vFlags patEnv ty fieldPats m =
     let fieldPats = fieldPats |> List.map (fun (fieldId, _, fieldPat) -> fieldId, fieldPat)
-    match BuildFieldMap cenv env true ty fieldPats m with
+    match BuildFieldMap cenv env false ty fieldPats m with
```

Comment:
`true`  -> ` { x with A... } `
`false`  -> `{ A = "" ; B = "" }"

Not sure why this is set to `true` but to me this should be always `false`